application/controllers/UserRoleController.php

```diff
@@ -242,6 +242,10 @@ public function actionViewRole(int $ptid)
      */
     public function actionRunExport($ptid)
     {
+        if (!Permission::model()->hasGlobalPermission('superadmin', 'read')) {
```

Comment:
403 ?

Reply:
Sorry, forgot to comment on it.
A 403 would have looked better to me as well, but the other stocks already had a flash message.